File: Лабораторная №0/python/4.py
from math import cos, sin, sqrt, log

x = float(input('Введите значение x: '))
y = (sqrt(x**5 + 4*sin(x)**2 - 8*log(x,2)))/(cos(x+6))
print(f'========\nОтвет: {round(y,3)}')

File: 4.pas
var x, y:real;

begin
  write('Введите x: ');
  read(x);
  y := (sqrt((power(x,5)) + 4*power(sin(x), 2) - 8*Log2(x)))/(cos(x+6));
  writeln(y:9:3);
end.